zerver/lib/message.py

```diff
@@ -1663,14 +1663,7 @@ def get_recent_private_conversations(user_profile: UserProfile) -> Dict[int, Dic
     return recipient_map
 
 
-def wildcard_mention_allowed(sender: UserProfile, stream: Stream, realm: Realm) -> bool:
-    # If there are fewer than Realm.WILDCARD_MENTION_THRESHOLD, we
-    # allow sending.  In the future, we may want to make this behavior
-    # a default, and also just allow explicitly setting whether this
-    # applies to a stream as an override.
-    if num_subscribers_for_stream_id(stream.id) <= Realm.WILDCARD_MENTION_THRESHOLD:
-        return True
-
+def wildcard_mention_allowed(sender: UserProfile, realm: Realm) -> bool:
```

Comment:
Let's rename this to something like `large_stream_wildcard_mention_policy_authorizes_user`, maybe with a docstring noting this is a helper for the `topic_wildcard_mention_allowed` and `stream_wildcard_mention_allowed functions -- I want to make clear it's obvious from the codebase that this shouldn't be used directly.

Reply:
Yeah, Renamed it to `wildcard_mention_policy_authorizes_user`. Avoiding the `large_stream` part as it doesn't make sense with topic wildcard in effect. 

`wildcard_mention_policy` is the setting name so the function name clearly conveys that whether that setting permits the user.

Added doctring.